devops/docker/watch.env

```diff
@@ -0,0 +1,2 @@
+UPLOAD_SITE_URL=http://hawiah:9091
```

Comment:
We should make the localhost, so it not specific to your machine

Reply:
Fixed